src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class Tribe__Events__Aggregator__Processes__Batch_Imports {
+	public function hook() {
```

Comment:
* :no_entry_sign: Missing doc comment for class Tribe__Events__Aggregator__Processes__Batch_Imports
* :no_entry_sign: Class name is not valid; consider Tribe_Events_Aggregator_Processes_Batch_Imports instead

<sub>hash: a1e41d9ddb2778071d5de39e6037fe8e</sub>